### PR TITLE
Undo step

### PR DESCRIPTION
**Pull request by dwagg**

### CODE REVIEW

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -956,7 +963,7 @@
                         </constraints>
                     </view>
                     <toolbarItems/>
-                    <navigationItem key="navigationItem" title="Test Run Index" id="Bjc-ta-NLl">
+                    <navigationItem key="navigationItem" title="Test Run" id="Bjc-ta-NLl">
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/Base.lproj/Main.storyboard:965]: I changed the title on the index page to match the back button on the step detail page. It also makes it less crowded on smaller screens. Not 100% if this is what we want to do though. Any input would be great.

**Review by JasonRitz — CHANGES_REQUESTED** [review]
The Input results in the pop up text box does not clear when the user hits "cancel Run"

**Review by JasonRitz — APPROVED** [review]

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepViewController.swift [hunk 2]
```diff
@@ -125,4 +125,23 @@ class TestStepViewController: UIViewController, UITextViewDelegate {
         }
     }
     
+    @IBAction func didTapResetStep(_ sender: Any) {
+        let clearAlert = UIAlertController(title: "Clear Step", message: "This step's status will be set to \"Not Run\" and the results will be cleared. Are you sure?", preferredStyle: UIAlertControllerStyle.alert)
+        
+        clearAlert.addAction(UIAlertAction(title: "Yes, I'm sure", style: .default, handler: {
+            (action: UIAlertAction!) in
+            //TODO reset the Results field for the step
```
Comment (lcc4) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepViewController.swift]: Does this still need to be done? (The TODO comment)

Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepViewController.swift]: Nope, I have updated this

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/Base.lproj/Main.storyboard (hunk 1 above)
Comment (lcc4) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/Base.lproj/Main.storyboard:965]: I think it is fine.  If Jama wants us to change it, we will.

**Pull request merged**